community.main/pom.xml

```diff
@@ -23,24 +23,9 @@
         <developerConnection>${project.scm.connection}</developerConnection>
     </scm>
 
-    <dependencies>
-        <dependency>
-            <groupId>org.kaazing</groupId>
-            <artifactId>community.license</artifactId>
-            <!-- We really want -->
-            <!--     <version>{project.version}</version> -->
-            <!-- but due to inheritance rules, this won't work. -->
-            <version>develop-SNAPSHOT</version>
```

Comment:
This is not used any more, since it is instead a dependency directly on the plugin itself instead of on the project.